xds/src/main/java/io/grpc/xds/SharedXdsClientPoolProvider.java

```diff
@@ -111,19 +115,17 @@ public ImmutableList<String> getTargets() {
     return ImmutableList.copyOf(targetToXdsClientMap.keySet());
   }
 
-
   private static class SharedXdsClientPoolProviderHolder {
     private static final SharedXdsClientPoolProvider instance = new SharedXdsClientPoolProvider();
   }
 
   @ThreadSafe
   @VisibleForTesting
-  static class RefCountedXdsClientObjectPool implements ObjectPool<XdsClient> {
+  protected class RefCountedXdsClientObjectPool implements ObjectPool<XdsClient> {
```

Comment:
`protected` is weird to see here, as `SharedXdsClientPoolProvider` is `final`. Revert back to package-private? (It doesn't actually change anything, but just confusing.)